cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/images/AddLaunchPermissionsToImageAction.java

```diff
@@ -136,7 +136,8 @@
                     .withUserGroupsString(userGroupsString)
                     .build();
 
-            return new QueryApiExecutor().execute(inputs, imageInputs);
+            Map<String, String> queryMapResult = new QueryApiExecutor().execute(inputs, imageInputs);
```

Comment:
@Tirla-Alin the lines 139 and 140 are the exact thing that was before... so, unless you have planes to use later the Map<String, String> queryMapResult I don't see the point of this modification.